src/libraries/Common/src/System/TimeProvider.cs

```diff
@@ -63,9 +60,10 @@ public DateTimeOffset GetLocalNow()
             }
 
             long localTicks = utcDateTime.Ticks + offset.Ticks;
-            if ((ulong)localTicks > (ulong)s_maxDateTicks)
+            const long MaxTicks = 3155378975999999999; // DateTime.MaxTicks
```

Comment:
Consider to make the constant `internal` and reuse. Also for the `0`s.

Reply:
`internal` wouldn't help in this case - this file needs to compile outside SPC.